nucypher/cli/actions/configure.py

```diff
@@ -110,3 +118,64 @@ def handle_invalid_configuration_file(emitter: StdoutEmitter,
         emitter.message(INVALID_JSON_IN_CONFIGURATION_WARNING.format(filepath=filepath))
         # ... sorry.. we tried as hard as we could
         raise  # crash :-(
+
+
+def collect_worker_ip_address(emitter: StdoutEmitter, network: str, force: bool = False) -> str:
+
+    # From environment variable  # TODO: remove this environment variable?
+    ip = os.environ.get(NUCYPHER_ENVVAR_WORKER_IP_ADDRESS)
```

Comment:
This feels like something that belongs in the config file, not in the environment.

Reply:
Although it wasn't introduced in this PR, so it is more of a note for the future, I guess.

Reply:
Yeah - I agree.  Needs to be removed and set into configuration in a follow-up.

Reply:
I didn't remember that we had this envvar. Any ideas what was the motivation behind it?

Reply:
I honestly have no idea.  I can remove it as part of this PR if you think it fits in here.  I noticed it's used in a few tests too.